front/src/modules/trainschedule/components/ManageTrainSchedule/AddTrainScheduleButton.tsx

```diff
@@ -51,21 +53,80 @@ const AddTrainScheduleButton = ({
     rollingStockId: simulationConf.rollingStockID,
   });
 
+  const [postTrainSchedule] =
+    osrdEditoastApi.endpoints.postTimetableByIdTrainSchedules.useMutation();
+  const [postPacedTrain] = osrdEditoastApi.endpoints.postTimetableByIdPacedTrains.useMutation();
+
   const createTrainSchedules = async () => {
-    const validTrainConfig = checkCurrentConfig(simulationConf, t, dispatch, rollingStock?.name);
+    const validTimetableItemConfig = checkCurrentConfig(
+      simulationConf,
+      t,
+      dispatch,
+      rollingStock?.name
+    );
 
-    // TODO Paced train : remove this when connectiing the back to add a paced train
-    if (isPacedTrainMode) {
-      if (validTrainConfig) {
-        setDisplayTrainScheduleManagement(MANAGE_TRAIN_SCHEDULE_TYPES.none);
-      }
-      return;
-    }
+    if (!validTimetableItemConfig) return;
 
-    if (validTrainConfig) {
-      const { timetableId, firstStartTime, trainCount, trainDelta, trainStep, baseTrainName } =
-        validTrainConfig;
+    const { timetableId, firstStartTime, trainCount, trainDelta, trainStep, baseTrainName } =
+      validTimetableItemConfig;
 
+    if (showPacedTrains) {
+      try {
```

Comment:
Move the try above the test showPacedTrains
```suggestion
    try {
      if (showPacedTrains) {
      ...
```

Reply:
I kept it like this to facilitate the moment we will have to delete the `trainSchedule` code in the `else` instead of merging all try/catch. Is that ok fo you ?

Reply:
Ok :)